### PR TITLE
Editing Pod disruption budget

### PR DESCRIPTION
**Pull request by alaudazzi**
This PR suggests a few edits.

Relates to [#63068](https://github.com/elastic/cloud/issues/63068).

### CODE REVIEW

**Comment by alaudazzi** [comment]
@barkbay I don't know why the file `/elasticsearch/init-containers-plugin-downloads.asciidoc` got automatically added to this PR. This file was already reviewed and merged in #3643.

File: docs/orchestrating-elastic-stack-applications/elasticsearch/init-containers-plugin-downloads.asciidoc [hunk 1]
```diff
@@ -27,6 +27,10 @@ spec:
             bin/elasticsearch-plugin install --batch repository-gcs
 ----
 
-You can also override the Elasticsearch container image to use your own image with the plugins already installed, as described in the <<{p}-custom-images,custom images doc>>. The <<{p}-snapshots,snapshots>> document has more information on both these options. The Kubernetes document on https://kubernetes.io/docs/concepts/workloads/pods/init-containers/[init containers] has more information on their usage as well.
+You can also override the Elasticsearch container image to use your own image with the plugins already installed, as described in the <<{p}-custom-images,custom images doc>>. For more information on both these options, you can check the <<{p}-snapshots,Create automated snapshots>> section and the Kubernetes documentation on https://kubernetes.io/docs/concepts/workloads/pods/init-containers/[init containers].
```
Comment (charith-elastic) [inline docs/orchestrating-elastic-stack-applications/elasticsearch/init-containers-plugin-downloads.asciidoc]:
```suggestion
You can also override the Elasticsearch container image to use your own image with the plugins already installed, as described in <<{p}-custom-images>>. For more information on both these options, you can check <<{p}-snapshots>> section and the Kubernetes documentation on https://kubernetes.io/docs/concepts/workloads/pods/init-containers/[init containers].
```

File: docs/orchestrating-elastic-stack-applications/elasticsearch/init-containers-plugin-downloads.asciidoc [hunk 2]
```diff
@@ -27,6 +27,10 @@ spec:
...
 
-The init container inherits the image of the main container image if one is not explicitly set. It also inherits the volume mounts as long as the name and mount path do not conflict. It also inherits the Pod name and IP address environment variables.
+The init container inherits:
+
+* The image of the main container image, if one is not explicitly set.
+* The volume mounts, as long as the name and mount path do not conflict.
```
Comment (charith-elastic) [inline docs/orchestrating-elastic-stack-applications/elasticsearch/init-containers-plugin-downloads.asciidoc]:
```suggestion
* The volume mounts from the main container unless a volume mount with the same name and mount path is present in the init container definition.
```

File: docs/orchestrating-elastic-stack-applications/elasticsearch/pod-disruption-budget.asciidoc [hunk 3]
```diff
@@ -8,12 +8,12 @@ endif::[]
 [id="{p}-{page_id}"]
 = Pod disruption budget
 
-A link:https://kubernetes.io/docs/tasks/run-application/configure-pdb/[Pod Disruption Budget] allows limiting disruptions on an existing set of Pods while the Kubernetes cluster administrator manages Kubernetes nodes.
-Elasticsearch makes sure some indices don't become unavailable.
+A link:https://kubernetes.io/docs/tasks/run-application/configure-pdb/[Pod Disruption Budget] (PDB) allows you to limit disruptions on an existing set of Pods, while the Kubernetes cluster administrator manages Kubernetes nodes.
```
Comment (charith-elastic) [inline docs/orchestrating-elastic-stack-applications/elasticsearch/pod-disruption-budget.asciidoc:11]:
```suggestion
A link:https://kubernetes.io/docs/tasks/run-application/configure-pdb/[Pod Disruption Budget] (PDB) allows you to limit the disruption to your application when its pods need to be rescheduled for some reason such as upgrades or routine maintenance work on the Kubernetes nodes.
```

File: docs/orchestrating-elastic-stack-applications/elasticsearch/pod-disruption-budget.asciidoc [hunk 4]
```diff
@@ -8,12 +8,12 @@ endif::[]
...
+Elasticsearch makes sure that some indices don't become unavailable.
```
Comment (charith-elastic) [inline docs/orchestrating-elastic-stack-applications/elasticsearch/pod-disruption-budget.asciidoc:12]: I don't think this line is necessary.

```suggestion

```

File: docs/orchestrating-elastic-stack-applications/elasticsearch/init-containers-plugin-downloads.asciidoc (hunk 1 above)
Comment (alaudazzi) [reply docs/orchestrating-elastic-stack-applications/elasticsearch/init-containers-plugin-downloads.asciidoc]: @charith-elastic the links formatting is weird in your suggestion. 
I propose to keep this version:

> You can also override the Elasticsearch container image to use your own image with the plugins already installed, as described in <<{p}-custom-images,custom images>>. For more information on both these options, you can check the <<{p}-snapshots,Create automated snapshots>> section and the Kubernetes documentation on https://kubernetes.io/docs/concepts/workloads/pods/init-containers/[init containers].

**Review by charith-elastic — APPROVED** [review]

**Comment by alaudazzi** [comment]
@charith-elastic I addressed the feedback you provided on `elasticsearch/init-containers-plugin-downloads.asciidoc` in a separate PR => #3647.  My apologies for the confusion.

**Pull request merged**